docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -41,6 +42,21 @@ class PackageConan(ConanFile):
         "with_foobar": True,
     }
 
+    @property
+    def _min_cppstd(self):
+        return 11
```

Comment:
```suggestion
        return 14
```
I guess these compilers are C++14, not C++11

Reply:
Nice! I've updated the other usages to be consistent.